templates/server/src/index.ts

```diff
@@ -33,7 +33,7 @@ try {
       console.log(`GraphQL Server is now running on http://localhost:${PORT}`);
       //{^isWorkspace//}
       if (firstStartInDevMode) {
-        opn(`http://localhost:${PORT}/api/swagger`);
+        opn(`http://localhost:${PORT}/graphql`);
```

Comment:
Should rather be `/graphiql`
```suggestion
        opn(`http://localhost:${PORT}/graphiql`);
```

Reply:
Jep, sorry - was in the mindset of https://github.com/sysgears/create-apollo-app/issues/70 :yum: